org.eclipse.winery.repository.rest/pom.xml

```diff
@@ -107,10 +107,9 @@
 			<scope>compile</scope>
 		</dependency>
 		<dependency>
-			<!-- Annotations only. We will switch to Swagger soon -->
-			<groupId>org.restdoc</groupId>
-			<artifactId>restdoc-api</artifactId>
-			<version>1.3.0</version>
+			<groupId>io.swagger</groupId>
+			<artifactId>swagger-jersey2-jaxrs</artifactId>
```

Comment:
Why jersey2 and not jersery1?

Reply:
Sorry, copy paste mistake, will be fixed shortly.